tests/cases/fourslash/quickInfoInheritDoc6.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference path="fourslash.ts" />
+
+// @Filename: quickInfoInheritDoc6.ts

```

Comment:
The names of these files (and their `@filename` comments) all contain `InheritDoc`, but none of them contain `@inheritdoc`